auth-web/src/components/auth/staff/DissolutionSchedule.vue

```diff
@@ -0,0 +1,296 @@
+          >
+            <label id="company-label">Schedule Summary</label>
+          </v-col>
+          <v-col
+            cols="12"
+            sm="6"
+          >
+            <span>
+              Moving <strong>{{ scheduleSummaryNumber }}</strong> businesses into D1 dissolution every
+              <strong>Tuesday</strong> at <strong>11:59 p.m</strong> Pacific Time.
+            </span>
+          </v-col>
+
+          <template v-if="!isEdit">
+            <v-col
+              cols="6"
+              sm="3"
+              class="text-right"
+            >
+              <v-btn
+                color="primary"
+                class="action-btn px-6"
+                @click="actionBtnClicked()"
+              >
+                {{ actionBtnText }}
+              </v-btn>
+              <span>
+                <v-menu
+                  v-model="menu"
+                  offset-y
+                  nudge-left="90"
+                >
+                  <template #activator="{ on }">
+                    <v-btn
+                      color="primary"
+                      class="more-actions-btn menu-btn px-3"
+                      v-on="on"
+                    >
+                      <v-icon v-if="menu">
+                        mdi-menu-up
+                      </v-icon>
+                      <v-icon v-else>
+                        mdi-menu-down
+                      </v-icon>
+                    </v-btn>
+                  </template>
+
+                  <v-list dense>
+                    <v-list-item @click="triggerEditOnOff()">
+                      <v-list-item-subtitle>
+                        <v-icon>mdi-pencil</v-icon>
+                        <span class="pl-2 edit-txt">Edit</span>
+                      </v-list-item-subtitle>
+                    </v-list-item>
+                  </v-list>
+                </v-menu>
+              </span>
+            </v-col>
+          </template>
+        </v-row>
+
+        <!-- Cancel/Save buttons -->
+        <template v-if="isEdit">
+          <v-divider class="mb-8 mt-8" />
+          <v-row no-gutters>
+            <v-col
+              cols="12"
+              class="text-right"
+            >
+              <v-btn
+                color="primary"
+                large
+                outlined
+                class="mr-3"
+                @click="triggerEditOnOff()"
+              >
+                Cancel
+              </v-btn>
+              <v-btn
+                color="primary"
+                large
+                @click="saveBtnClicked()"
+              >
+                Save
+              </v-btn>
+            </v-col>
+          </v-row>
+        </template>
+      </article>
+    </section>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Vue } from 'vue-property-decorator'
+import { mapActions } from 'pinia'
+import { useStaffStore } from '@/stores/staff'
+
+@Component({
+  methods: {
+    ...mapActions(useStaffStore,
```

Comment:
We usually use the `@Getter` and `@Action` in our entities codebases but in here, I'm just following the coding practice of relationships to stay uniform across all files 😃

Reply:
I was going to comment on this, not because I think mapActions() is bad, but because you also need to declare the method signatures on lines 158-161. DRY if possible.